cypress/e2e/awx/overview/awx-overview.cy.ts

```diff
@@ -3,8 +3,8 @@ import { Inventory } from '../../../../frontend/awx/interfaces/Inventory';
 import { Job } from '../../../../frontend/awx/interfaces/Job';
 import { Project } from '../../../../frontend/awx/interfaces/Project';
 import { awxAPI } from '../../../support/formatApiPathForAwx';
-
-describe('AWX Overview', () => {
+// FLAKY_06_13_2024
+describe.skip('AWX Overview', () => {
```

Comment:
I don't believe we will need to skip this test once this [PR](https://github.com/ansible/ansible-ui/pull/2521) merges.